### DISCUSSION TITLE
Add support to superscript and subscript

### DISCUSSION THREAD

**Discussion by foxtrotcz** [tailwindlabs/tailwindcss#4208] [Ideas]
Hello,
sometimes I need to write units like m<sup>2</sup>.
I would like if Tailwind added support for superscript and subscript.
https://helpx.adobe.com/fonts/using/open-type-syntax.html#sups
Thanks.

**Comment by cckalen** [comment]
In the context of block-level typographic control, leveraging font-variant-position utilities is feasible. 

https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant-numeric

`font-variant-numeric: ordinal;`

However, the specific typeface in use might trigger substitutions of alternate glyphs when these utilities are applied. For scenarios requiring  more advanced typographic features, integrating Tailwind utility classes would be effective.  I found this to be helpful;
https://tailwindcss-opentype.netlify.app/

**Comment by kbegiedza** [comment]
You can try using `sub` and `sup` html tags.

```html
<p>
  Some text that I would like to<sub>sub</sub> or some<sup>sup</sup> to <br/>
  Create stuff like:<br/>
  m<sup>2</sup> or even nested k<sup>(n+1)<sup>2</sup></sup>
</p>
```
![image](https://github.com/user-attachments/assets/9795bfd1-f50d-4bb0-bfdb-cd6fda0fcc55)

https://play.tailwindcss.com/WC5HOPb8vk
